libs/sdk-ui-dashboard/src/model/_staging/dashboard/getDashboardFilterRef.ts

```diff
@@ -0,0 +1,33 @@
+// (C) 2021 GoodData Corporation
+import { IDashboardFilter } from "@gooddata/sdk-ui-ext";
+import { UnexpectedSdkError } from "@gooddata/sdk-ui";
+import {
+    ObjRef,
+    isNegativeAttributeFilter,
+    isPositiveAttributeFilter,
+    isAbsoluteDateFilter,
+    isRelativeDateFilter,
+} from "@gooddata/sdk-model";
+
+/**
+ * Gets dashboard filter ref.
+ * Display form for attribute filter, or date data set ref for date filter.
+ *
+ * @internal
+ * @param filter - dashboard filter to obtain ref for
+ * @returns filter ref
+ */
+export function getDashboardFilterRef(filter: IDashboardFilter): ObjRef {
+    if (isPositiveAttributeFilter(filter)) {
```

Comment:
Nitpick: could be simplified using `filterObjRef` from sdk-model that does similar thing.